package.json

```diff
@@ -9,7 +9,7 @@
     "grunt-contrib-concat": "~0.4.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-cssmin": "~0.9.0",
-    "grunt-contrib-jshint": "~0.6.0",
+    "grunt-eslint": "^7.0.1",
```

Comment:
Mm, I wondered on this as well. I went off of what eregs did https://github.com/cfpb/regulations-site/pull/684/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R20 I guess if linting was part of a deployment process a break in the eslint plugin would be picked pretty quick? It seems like npm has gone more in a ^ direction overall... http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/

I don't know enough about the pitfalls here to really know which way to go. @ascott1 Do you have input here?